src/benchmark/join_order_benchmark.cpp

```diff
@@ -113,5 +114,27 @@ int main(int argc, char* argv[]) {
   auto query_generator =
       std::make_unique<FileBasedQueryGenerator>(*benchmark_config, query_path, non_query_file_names, query_subset);
 
-  BenchmarkRunner{*benchmark_config, std::move(query_generator), std::move(table_generator), context}.run();
+  auto benchmark_runner =
+      BenchmarkRunner{*benchmark_config, std::move(query_generator), std::move(table_generator), context};
+
+  std::cout << "- Adding indexes to SQLite... " << std::flush;
+
+  if (benchmark_config->verify) {
+    // Add indexes to SQLite. This is a hack until we support CREATE INDEX ourselves and pass that on to SQLite
```

Comment:
Comment that without this SQLite cannot even finish some queries and that we therefore cannot avoid adding the indices?

Reply:
From the diff, the requested comment is still missing. Or do you think it isn't helpful? I think it'd help me...

Reply:
I had "Without this, SQLite would never finish." in there, but a couple of lines below. Moved it up.